.github/workflows/desktop.yml

```diff
@@ -160,7 +160,7 @@ jobs:
         run: |
           EXTRA_CMAKE_ARGS="-DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded\$<\$<CONFIG:Debug>:Debug>"
           EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -DPKG_CONFIG_EXECUTABLE=${CMAKE_BUILD_DIR}/vcpkg_installed/x64-windows/tools/pkgconf/pkgconf.exe"
-          EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -D WITH_BLUETOOTH=OFF" # Not yet properly tested/supported and we have a conflict of double defined symbol in QtBluetooth/QtMultimedia
+          EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -DWITH_BLUETOOTH=OFF" # Not yet properly tested/supported and we have a conflict of double defined symbol in QtBluetooth/QtMultimedia
```

Comment:
That was on purpose (I started to use the space more often for readability) but you can still convince me of the opposite ;-)

Reply:
It's pulled in for app and qmltest targets because in a qml there is still an `import QtBluetooth` somewhere, working on it